### PR TITLE
Update oom_score_adjusted_by_redis to oom_score_adjusted_by_valkey in server.c

### PR DESCRIPTION
**Pull request by Shivshankar-Reddy**
Update oom_score_adjusted_by_redis to oom_score_adjusted_by_valkey in server.c

### CODE REVIEW

File: src/server.c [hunk 1]
```diff
@@ -267,7 +267,7 @@ void exitFromChild(int retcode) {
 /*====================== Hash table type implementation  ==================== */
 
 /* This is a hash table type that uses the SDS dynamic strings library as
- * keys and redis objects as values (objects can hold SDS strings,
+ * keys and valkey objects as values (objects can hold SDS strings,
```
Comment (CharlesChen888) [inline src/server.c]: Since we already renamed "redisObject" to "serverObject", perhaps "server objects" here or just "objects"?

Comment (Shivshankar-Reddy) [reply src/server.c]: Looks unstable is already updated as you suggested.

File: src/server.c [hunk 2]
```diff
@@ -2769,6 +2769,7 @@ void initServer(void) {
      * no explicit limit in the user provided configuration we set a limit
      * at 3 GB using maxmemory with 'noeviction' policy'. This avoids
      * useless crashes of the instance for out of memory. */
+
```
Comment (CharlesChen888) [inline src/server.c]: Invalid line addition.

Comment (Shivshankar-Reddy) [reply src/server.c]: Removed th eline and updated diff, Thanks!

**Review by CharlesChen888 — APPROVED** [review]

**Comment by Shivshankar-Reddy** [comment]
@zuiderkwast Can you review this when you have sometime?

File: src/server.c [hunk 3]
```diff
@@ -6698,7 +6698,7 @@ void loadDataFromDisk(void) {
 void serverOutOfMemoryHandler(size_t allocation_size) {
     serverLog(LL_WARNING,"Out Of Memory allocating %zu bytes!",
         allocation_size);
-    serverPanic("Redis aborting for OUT OF MEMORY. Allocating %zu bytes!",
+    serverPanic("Valkey aborting for OUT OF MEMORY. Allocating %zu bytes!",
```
Comment (zuiderkwast) [inline src/server.c]: Please separate log messages (like this one) into separate PRs. They are potentially breaking changes. We need to keep track of them carefully for release notes, etc.

File: src/server.c [hunk 4]
```diff
@@ -7033,7 +7033,7 @@ int main(int argc, char **argv) {
                 exit(0);
             } else {
                 fprintf(stderr,"Please specify the amount of memory to test in megabytes.\n");
-                fprintf(stderr,"Example: ./redis-server --test-memory 4096\n\n");
+                fprintf(stderr,"Example: ./valkey-server --test-memory 4096\n\n");
```
Comment (zuiderkwast) [inline src/server.c]: This is the output of a program that we might want to backport to 7.2.x release, without all the other changes. I would prefer this in separat PR as well (or together with changes of the same kind).

**Review by zuiderkwast — COMMENTED** [review]
Thanks!

All seems good, but I want to separate into separate PRs, for various reasons. That's why it was separated in #25 into separate categories.

1. internals (variable names and comments)
2. log messages
3. other output like usage

File: src/server.c (hunk 4 above)
Comment (Shivshankar-Reddy) [reply src/server.c]: Sure, updated the diff by removing these line chnages, and now its only variable rename.

**Review by enjoy-binbin — APPROVED** [review]

**Pull request merged**